nat-lab/tests/telio.py

```diff
@@ -937,23 +947,6 @@ async def clear_system_log(self) -> None:
                     f"Clear-EventLog -LogName {log_name}",
                 ]).execute()
 
-    async def get_log_lines(self, regex: Optional[str] = None) -> List[str]:
```

Comment:
This isn't used anymore, but maybe we can keep this as a util for any future test ? Or is there another alternative for this now ?

Reply:
IMHO, whenever we actually have a need for it, we can add it. Right now there are no usages of it.